docs/source/using/experimental-features.adoc

```diff
@@ -151,3 +151,40 @@ To get the status of the DNS server:
 ----
 $ minishift dns status
 ----
+
+[[run-against-an-existing-machine]]
+== Run against an Existing Machine
+
+{project} can be run against an existing machine using `vm-driver` as `generic` along with it options.
+To run against an existing machine you first need to configure your (remote/local/VM) machine.
+
+[NOTE]
+====
+This features as of now tested for Fedora/CentOS and RHEL machines.
+====
+
+We are assuming that `10.1.1.1` is the IP of Machine where you want to run {project} against.
+
+On your host execute as follows:
+
+----
+Host$ ssh-copy-id <user>@10.1.1.1  // Make sure user have sudo access without password or use root
+Host$ ssh <user>@10.1.1.1  // Make sure this login without password
+----
+
+Configuring your remote Machine as follows:
+
+----
+Remote_VM$ yum install -y docker net-tools
+Remote_VM$ systemctl start docker
+Remote_VM$ systemctl enable docker
+----
+
+You also need to allow `2376`, `8443` port on the machine from firewall to have communication from the host.
+
+Run {project} against remote Machine as follows:
+
+----
+Host$ export MINISHIFT_ENABLE_EXPERIMENTAL=y
+Host$ minishift start --vm-driver generic --remote-ip-address 10.1.1.1 --remote-ssh-user <username> --remote-ssh-key <Private_key_path_which_used_for_ssh_copy_id>
```

Comment:
`Private_key_path_which_used_for_ssh_copy_id` is not clear enough. May be we should add a note explaining why is this required.